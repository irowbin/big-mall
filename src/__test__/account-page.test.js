import { shallow } from 'enzyme'
import AccountPage from '../pages/account/account.component'

describe('AccountPage component', () => {
  it('should match snapshot', () => {
    expect(shallow(<AccountPage />)).toMatchSnapshot()
  })
})
