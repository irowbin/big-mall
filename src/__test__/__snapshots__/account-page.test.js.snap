// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountPage component should match snapshot 1`] = `ShallowWrapper {}`;
